### ISSUE TITLE
DKIM keys aren't generated

### ISSUE THREAD

**Issue by neskhodovskiy** [Mailu/Mailu#1892]
Hello, 

I have set up mailu and it's generally working fine, I can add domains and users and receive mail, but I'm unable to proceed with DKIM keys generation.

My steps:
Click on "details" icon for the domain 
- I see a table with three lines: DNS MX entry, DNS SPF entries, DNS client auto-configuration (RFC6186) entries. NO DKIM

Click "Regenerate Keys"
- I see a confirmation prompt

Click "Confirm"
- A POST request is sent to https://mail.example.com/admin/ui/domain/genkeys/example.com and results in a 302 redirect back to the details page for the domain. But the details still show only the 3 above mentioned lines. DKIM keys do not appear. 

I don't know if it's relevant but the permissions to the /mailu/dkim directory on the host machine are

```
drwxr-xr-x  2 root   root     4096 Jul 24 17:51 dkim
```

I tried setting them temporarily to 777 but that didn't help so I restored them to 755. This directory remains empty before and after the request.

### Environment
 - [x] docker-compose on Ubuntu 20.04

### Versions
Mailu version is 1.7 

## Logs
I've been watching the logs of the antispam and admin containers during the regeneration request. Neither container generates any new log lines at the moment of request. 

Please let me know how I can help debug the issue or if there's a command I can run in the container manually to generate keys for DKIM. Thanks!

**Comment by ghostwheel42** [comment]
@neskhodovskiy Are you sure you are using version 1.7?
I can't replicate this with 1.7 (or 1.8) - but I can replicate it in master.

**Comment by neskhodovskiy** [comment]
Yes I'm sorry I misunderstood the version I'm using. 

I looked up and the image names indeed say "master".

How do I switch to a stable version?

**Comment by neskhodovskiy** [comment]
Nevermind I figured it out. Just added this line to `.env`:

MAILU_VERSION=1.7

and re-run

`docker-compose down && docker-compose up -d --build`

(After this, the application started showing an error page when I tried to log in as an admin. I had to remove the /mailu data directory and re-create it from scratch. Luckily, I didn't have much to lose at this point. Now it's working and DKIM keys can be properly generated). 

Sorry for the confusion and thanks for the hint!
